SU-HKKU/cardinal_service_profile
# CSD-232 Added field to spotlight content type like opportunities

# READY FOR REVIEW

# Summary
- Added some fields to the spotlight content type so that they are similar to the opportunities
- Fixed the opportunities view because the settings form the related block changed the settings for the all opportunity list block

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. `composer require SU-HKKU/cardinal_service_profile:dev-CSD-232`
1. `blt drupal:sync --site=cardinalservice`
1. view the `/opportunities` page and verify its looks back to normal
1. verify the fields are on the spotlight content type.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
